requirements.txt

```diff
@@ -22,6 +23,7 @@ pip>=21.0,<22.3
 python-slugify==4.0.1
 pyyaml==6.0
 requests==2.28.1
+sqlalchemy==1.4.40
```

Comment:
This is a dependency of `recorder`, and not a core dependency?
I don't see any part of core using this. Could you elaborate on this addition?

Reply:
If you don't have this installed, Home Assistant immediately quits on startup with an error about not having it.